src/core/ext/xds/xds_bootstrap.h

```diff
@@ -83,23 +95,33 @@ class XdsBootstrap {
   struct Authority {
     std::string client_listener_resource_name_template;
     std::vector<XdsServer> xds_servers;
+
+    static const JsonLoaderInterface* JsonLoader();
   };
 
   // Creates bootstrap object from json_string.
-  // If *error is not GRPC_ERROR_NONE after returning, then there was an
-  // error parsing the contents.
-  static std::unique_ptr<XdsBootstrap> Create(absl::string_view json_string,
-                                              grpc_error_handle* error);
+  static absl::StatusOr<XdsBootstrap> Create(absl::string_view json_string);
 
   // Do not instantiate directly -- use Create() above instead.
-  XdsBootstrap(Json json, grpc_error_handle* error);
+  XdsBootstrap() = default;
+
+  // Copyable.
+  XdsBootstrap(const XdsBootstrap&);
```

Comment:
Do we need copyable? (the copy constructors for the maps and vectors are adding a fair amount of bloat here)

Reply:
No, we don't.  Removed.